README.md

```diff
@@ -7,30 +7,29 @@ Look at the [nuxt 3 documentation](https://v3.nuxtjs.org) to learn more.
 Make sure to install the dependencies:
 
 ```bash
-# yarn
-yarn install
+npm install
 ```
 
 ## Development Server
 
-Start the development server on http://localhost:3000
+Start the development server on http://localhost:3000/What-am-I-listening-to
 
 ```bash
-yarn run dev
+npm start
 ```
 
 ## Production
 
 Build the application for production:
 
 ```bash
-yarn run build
+npm run build
```

Comment:
Ich glaube deploy baut es für production. Postdeploy deployed es dann. So wie du es in der readme beschrieben hast macht es aber sinn. Willst du es auch in der package.json anpassen?